librz/include/rz_bin.h

```diff
@@ -459,6 +459,8 @@ typedef struct rz_bin_plugin_t {
 	int (*get_offset)(RzBinFile *bf, int type, int idx);
 	char *(*get_name)(RzBinFile *bf, int type, int idx, bool simplified);
 	ut64 (*get_vaddr)(RzBinFile *bf, ut64 baddr, ut64 paddr, ut64 vaddr);
+	char* (*section_type_to_string)(ut64 type);
```

Comment:
Just to understand... In the future we might have as well a `segment_type_to_string` and `segment_flags_to_string`, right?

Reply:
Right now the type of segment seems to be mixed up with its name...

Reply:
> Just to understand... In the future we might have as well a `segment_type_to_string` and `segment_flags_to_string`, right?

That would be a great addition for uniformity, also with the section permissions, I saw comments to make it a separate function and I was planning to look at that next.